sectionproperties/analysis/fea.py

```diff
@@ -58,19 +64,30 @@ def geometric_properties(self):
         for gp in gps:
             # determine shape function, shape function derivative and jacobian
             (N, _, j) = shape_function(self.coords, gp)
+            # log.log(level=logging.DEBUG, msg=f"N: {N}\nj: {j}")
```

Comment:
Remove temporary log/commented out code?

Reply:
Darn it! I did a Ctrl-F in cross_sections.py yesterday but not in fea.py. Will do!

Reply:
Removed!